### ISSUE TITLE
Issues in systemd man pages

### ISSUE THREAD

**Issue by hmartink** [systemd/systemd#35751]
### systemd version the issue has been seen with

257-2

### Used distribution

various

### CPU architectures issue was seen on

None

### Component

other

### Expected behaviour you didn't see

Dear systemd maintainer,
attached you find the recently found issues in the man pages of systemd. I hoped I masked out all issues you do not agree with (in the previous rounds), but please accept my apologies if I missed one or two, this is not on purpose.

Thanks for handling them and a happy holiday season (if applicable)

### Unexpected behaviour you saw

[bugreport.systemd.txt](https://github.com/user-attachments/files/18244376/bugreport.systemd.txt)

**Comment by TriMoon** [comment]
@hmartink , Because your attachment contains a LOT of pieces that need to be addressed, i think it is best to post the separate issues with accompanied explanation of what is exactly wrong.

That way each one can be tracked easier by the maintainers, and not require downloading the attachment everytime someone works on them and take notes inside it...

**Comment by hmartink** [comment]
The last times the format was fine, and a single person on your side handled the issues, which IMHO was quite efficient, as we could discuss them looking at the whole body of systemd pages (e.e. reocurring issues). I can of course paste them in the form, but last time (about one month ago) the attached file was handled fine. Opening separate issues for each page (?) increases the workload significantly on my side. And I have > 100 projects where I report bugs to. (If it can be automated, like in projects which accept issue reports by e-mail, I could script it of course, but with web based form this is  a PITA).

And the explanations are as they have been like the last 11 times. If there are any questions, I do respond to them, but hopefully most of them are self-explanatory (and my "style" is already known).

**Comment by TriMoon** [comment]
> and a single person on your side handled the issues

Note that i am not part of the team, i'm just an independent person on the internet like you :wink:

**Comment by keszybz** [comment]
Yeah, a single report is fine. I'm one of the people who handled those reports in the past. Having separate issues would increase the workload on my side too.

**Comment by yuwata** [comment]
Yeah, I agree.
